test/mocha.opts

```diff
@@ -1,3 +1,5 @@
+test/chai-extensions.js
 test/specs/**/*.js
+--timeout 3000
```

Comment:
in the next lines we use abbreviation flags, could you change them all then? Or use `-t`?